db/schema.rb

```diff
@@ -576,7 +576,7 @@
     t.integer  "created_by_id"
   end
 
-  add_index "spree_orders", ["completed_at"], :name => "index_spree_orders_on_completed_at"
```

Comment:
This index is redundant because of PostgreSQL's left-most index columns usage.

As described in https://www.celerity.com/how-to-design-sql-indexes/:

> Additionally, multi-column indexes can still be used by the query planner even when all of the columns in the index aren’t constrained upon. For example, if you define a multi-column index against a user id, a start time, and an end time (in that order), that index can be used for queries against the user id alone, as well as the user id & start time together. The order of columns in the index definition matter: so long as your subset of columns follows the index definition left-to-right, that multi-column index is effectively and efficiently used.